### DISCUSSION TITLE
WPA enterprise Wifi connectivity i.e. eduroam

### DISCUSSION THREAD

**Discussion by pluralitas** [meshtastic/firmware#3547] [Ideas]
It would be nice to be able to connect to e.g. eduroam or other WPA enterprise networks and using MQTT to link between mesh network of different campuses.

**Comment by fifieldt** [comment]
It looks like at least the ESP Wifi supports this: https://github.com/espressif/arduino-esp32/tree/master/libraries/WiFi/examples/WiFiClientEnterprise

For Meshtastic this would involve adding several new entries to the protobufs.
